src/cashlink.ts

```diff
@@ -0,0 +1,62 @@
+import Vue from 'vue';
+import { BrowserDetection } from '@nimiq/utils';
+import App from './CashlinkApp.vue';
+import store from './store';
+// import { init as initSentry, captureException } from '@sentry/browser';
+// import { Vue as SentryVueIntegration } from '@sentry/integrations';
+import VueRaven from 'vue-raven'; // Sentry.io SDK
+// @ts-ignore
+import IqonsSvg from '@nimiq/iqons/dist/iqons.min.svg';
+import Config from 'config';
+
+if (window.hasBrowserWarning) {
+    throw new Error('Exeution aborted due to browser warning');
+}
+
+if ((BrowserDetection.isIOS() || BrowserDetection.isSafari()) && 'serviceWorker' in navigator) {
+    // Register service worker to strip cookie from requests
+    navigator.serviceWorker.register('/ServiceWorker.js', {
+        scope: '/',
+    }).then((reg) => {
+        console.debug(`Service worker has been registered for scope: ${reg.scope}`);
+    }).catch((error) => {
+        console.error(`Service worker installation failed`);
+        throw error;
+    });
+}
+
+Vue.config.productionTip = false;
+
+// Set up Identicon SVG file path
+if (IqonsSvg[0] === '"') {
+    // @ts-ignore
+    self.NIMIQ_IQONS_SVG_PATH = IqonsSvg.substring(1, IqonsSvg.length - 1);
+} else {
+    // @ts-ignore
+    self.NIMIQ_IQONS_SVG_PATH = IqonsSvg;
+}
+
+if (Config.reportToSentry) {
+    // initSentry({
```

Comment:
Yes, pending https://github.com/nimiq/hub/pull/310. It's commented here because I was waiting for that PR to be merged to then replace it here. I didn't think #310 would not be approved at this point...